oop_hw3_v2/wagon.py

```diff
@@ -0,0 +1,17 @@
+from aqa_course.oop_hw3_v2.train import Train
+
+
+class Wagon(Train):
+    def __init__(self):
+        super().__init__()
+
+    def add_passenger(self, passenger: str):
+        return self.list_of_passengers.append(passenger)
+
+    def __len__(self, arg):
+        if arg == self:
+            return len(self.list_of_wagons)
+        elif arg == Wagon:
+            return len(self.list_of_passengers)
```

Comment:
When printing a wagon to the console, I want to see the following "[n]" where n is the number of the wagon.

Where is implementation of this part of task?